README.md

```diff
@@ -4,5 +4,29 @@ AppVeyor: [![AppVeyor](https://ci.appveyor.com/api/projects/status/fxhto3omtehio
 
 Travis:   [![Travis](https://travis-ci.org/aspnet/dnx-watch.svg?branch=dev)](https://travis-ci.org/aspnet/dnx-watch)
 
-This project is part of ASP.NET 5. You can find samples, documentation and getting started instructions for ASP.NET 5 at the [Home](https://github.com/aspnet/home) repo.
+With `dnx --watch`, when a file changed, dnx would exit and you had to restart it manually.
```

Comment:
I recommend starting the readme with a brief statement about what the project in this repo does. Something like "Use `dnx-watch` to automatically restart an application when its source code changes."